lib-compose/src/main/java/com/what3words/components/compose/maps/W3WMapComponent.kt

```diff
@@ -107,18 +106,22 @@ fun W3WMapComponent(
         onMapViewPortProvided = mapManager::setMapViewPort,
         onRecallClicked = {
             mapState.selectedAddress?.latLng?.let {
-                mapManager.moveToPosition(
-                    coordinates = W3WCoordinates(it.lat, it.lng),
-                    animate = true
-                )
+                coroutineScope.launch {
+                    withContext(Main) {
```

Comment:
We can use coroutineScope.launch(Main) without withContext(Main) for cleaner and efficient